src/addBusinessDays/test.ts

```diff
@@ -103,11 +103,53 @@ describe('addBusinessDays', function () {
       assert.deepStrictEqual(result, new Date(2022, 0 /* Jan */, 21))
     })
 
-    it('can override business days with exceptions', function () {
+    it('can include business days with true exceptions', function () {
```

Comment:
I added more tests for true exceptions, which were failing previously.

Reply:
question: A lot of tests are added here, but not in `subBusinessDays`. How confident are we that all these cases will `just work` in the opposite direction?